src/View/Fight/selectAttack.html.twig

```diff
@@ -0,0 +1,80 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}FightPage - {{parent()}}
+{% endblock %}
+
+{% block stylesheet %}
+    {{parent()}}
+    <link rel="stylesheet" type="text/css" href="/assets/css/battle.css"/> 
+{% endblock %}
+
+{% block content %}
+<main class="fight">
+
+    <section class="battle">
+        <img class="player" src="/assets/images/profile/{{sessionUser ? sessionUser.avatar : 'img_avatar.png'}}">
+        <img class="versus" src="/assets/images/vs-index.png">
+        <img class="player" src="/assets/images/profile/{{opponentUser ? opponentUser.avatar : 'img_avatar.png'}}">
+    </section>
+
+    <section class="round">
+        <h1> ROUND 1 </h1>
```

Comment:
should be incremented dynamically, if you don't hae time to do it, remove the number to only let "Round" as a title

Reply:
i change for ROUND, but it is ali who will feed the round once the branch merges